nbuild/stdenv/install.py

```diff
@@ -21,43 +22,56 @@ def make_symlink(src, dst):
     os.symlink(src, dst)
 
 
-def make_cp(*src, dest, root='', args=''):
+def make_cp(*files, dest, root=''):
     package = current_build().current_package
-    for filename in src:
-        path = f'{package.install_dir}/{root}/{filename}/'
-        cmd(f'cp {args} {path} {dest}')
+    for filename in files:
+        path = f'{package.install_dir}/{root}/{filename}'
+        shutil.copy2(path, dest)
 
 
-def make_mkdir(dir, args=''):
+def make_mkdir(dir):
     package = current_build().current_package
     path = f'{package.install_dir}/{dir}/'
-    cmd(f'mkdir {args} {path}')
+    if not os.path.exists(path):
+        os.makedirs(path)
 
 
-def make_rm(*files, root='', args=''):
+def make_rm_file(*files, root=''):
```

Comment:
I feel like if the name of the function includes what it removes (`_file`) then it should be `_files` instead